### PR TITLE
Adding Argo CD 1.2.5 initial stack

### PR DESCRIPTION
**Pull request by wadenick**
Initial Argo CD stack, versioned 1.2.5 (latest stable release). 

Some basic Getting Started / test instructions are included in README.md.

### CODE REVIEW

File: stacks/argocd/render.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```
Comment (paulanunda) [inline stacks/argocd/render.sh]: this file can be deleted along w/ stacks/argocd/yaml/argocd.yaml

File: stacks/argocd/deploy-local.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+kind: Namespace
+metadata:
+  name: argocd
+EOF
+
+# set kubectl namespace
+kubectl config set-context --current --namespace=argocd
+
+# deploy argocd
+kubectl apply -f "$ROOT_DIR"/stacks/argocd/yaml/argocd.yaml
```
Comment (paulanunda) [inline stacks/argocd/deploy-local.sh]: This should point to your src YAML: `"$ROOT_DIR"/src/argocd/1.2.5/argocd.yaml`

File: stacks/argocd/deploy-local.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (paulanunda) [inline stacks/argocd/deploy-local.sh]: Don't forget to patch the `svc`

File: stacks/argocd/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+# deploy argocd
+kubectl apply -f https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/src/argocd/1.2.5/argocd.yaml
+
+# add the DigitalOcean LBaaS LoadBalancer
+kubectl patch svc argocd-server -n argocd -p '{"spec": {"type": "LoadBalancer"}}'
+
+
+# ensure services are running
+kubectl rollout status deployment/argocd
```
Comment (paulanunda) [inline stacks/argocd/deploy.sh]: This should be:
```
kubectl get deployments -o custom-columns=NAME:.metadata.name | tail -n +2 | while read -r line
do
  kubectl rollout status deployment/"$line"
done
```

File: stacks/argocd/deploy-local.sh [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# ensure services are running
+kubectl rollout status deployment/argocd
```
Comment (paulanunda) [inline stacks/argocd/deploy-local.sh]: This should be:
```
kubectl get deployments -o custom-columns=NAME:.metadata.name | tail -n +2 | while read -r line
do
  kubectl rollout status deployment/"$line"
done
```

File: stacks/argocd/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Login using the Argo CD CLI
+
+Login as the admin user. The initial password is autogenerated to be the pod name of the Argo CD API server. This can be retrieved with the command:
+
+`kubectl get pods --all-namespaces`
+
+You'll see output similar to the following: 
+
+`yourhostname:.kube yourusername$ kubectl get pods --all-namespaces
```
Comment (paulanunda) [inline stacks/argocd/README.md]: use triple ticks to preserve text formatting

File: stacks/argocd/deploy.sh (hunk 4 above)
Comment (wadenick) [reply stacks/argocd/deploy.sh]: ¯\_(ツ)_/¯ it was your autogenerated line 

Will update, thanks

File: stacks/argocd/deploy-local.sh (hunk 3 above)
Comment (wadenick) [reply stacks/argocd/deploy-local.sh]: Doh

File: stacks/argocd/render.sh (hunk 1 above)
Comment (wadenick) [reply stacks/argocd/render.sh]: +1

File: stacks/argocd/deploy.sh (hunk 4 above)
Comment (paulanunda) [reply stacks/argocd/deploy.sh]: noted, i'll update the auto-generator

File: stacks/argocd/deploy-local.sh (hunk 2 above)
Comment (wadenick) [reply stacks/argocd/deploy-local.sh]: Corrected, I hadn't saved a few things in VS Code *facepalm* 🤦‍♀️

File: stacks/argocd/deploy.sh (hunk 4 above)
Comment (wadenick) [reply stacks/argocd/deploy.sh]: updated for deployment here 👍

File: stacks/argocd/README.md (hunk 6 above)
Comment (wadenick) [reply stacks/argocd/README.md]: thanks - made this change

**Pull request merged**